dxr/query.py

```diff
@@ -29,6 +30,34 @@ def direct_searchers(plugins):
     return [searcher for searcher, _ in sortables]
 
 
+def _construct_query(filters):
```

Comment:
Can we name this more informatively? Right now, "query" could be an ES query or a Query object (and Query objects are, after all, constructed). `_make_es_query`?